### DISCUSSION TITLE
Random Patches và Random Subspaces

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#46] [Terminology] [answered]
Trong phần Ensemble learning trong chương 7 có phương pháp `Random Patches` và `Random Subspaces`.
* Random Patches là phương pháp mà mình lấy mẫu dựa trên cả mẫu dữ liệu huấn luyện (instances) và đặc trưng (features)
* Random Subspaces là phương pháp lấy mẫu theo đặc trưng (features).

Random Subspaces thì mình nghĩ dịch là `Không gian con Ngẫu nhiên` còn `Random Patches` dịch sao cho hợp lý nhỉ?:
* Patch Ngẫu nhiên
* Miếng vá Ngẫu nhiên
* Mảnh Ngẫu nhiên
* ...

Mention @mlbvn/handson-ml2

**Comment by rootonchair** [comment]
`Miếng vá Ngẫu nhiên` hoặc `Phần vá Ngẫu nhiên` em thấy ok. Dù sao nó cũng đúng với cách nhìn của phương pháp

**Reply by ngcthuong** [reply]
Patch chỉ nên dịch là mảnh thôi. Không nên cho từ "vá" vào.

**Comment by minhduc0711** [comment, marked as answer]
chốt nhé mọi ng
* `random patch`: `patch ngẫu nhiên`
* `random subspace`: `không gian ngẫu nhiên`

**Reply by rootonchair** [reply]
Riêng từ `mảnh` rất là general. Em thấy nếu chưa kiếm từ phù hợp xong thì khi dịch mình có thể để nguyên rồi sửa lại sau được không @minhduc0711 ?

**Reply by minhduc0711** [reply]
à ok sorry, mình tưởng mọi người thấy ổn rồi. Vậy mình để lại là `patch ngẫu nhiên` nhé?

**Reply by rootonchair** [reply]
Ok ạ

**Reply by ngcthuong** [reply]
Đợi dịch xong mình soát lại một lượt. Cá nhân mình thấy hiện tại có quá nhiều từ khóa mượn.
